warehouse/upload.go

```diff
@@ -2063,3 +2073,18 @@ func (job *UploadJobT) UpdateLocalSchema(schema warehouseutils.SchemaT) error {
 // 	}
 // 	return batchedStagingFiles
 // }
+
+func GetWarehouseDestinationSuccessCount(dbHandle *sql.DB, destinationID string) (successfulExportCount int, err error) {
+	query := fmt.Sprintf("select count(*) from %s where destination_id='%s' ", warehouseutils.WarehouseUploadsTable, destinationID)
+	rows, err := dbHandle.Query(query)
```

Comment:
If it is successful export count, query should contain `status = 'exported_data'`